src/types/index.d.ts

```diff
@@ -91,6 +92,7 @@ export type NewsStoryType = PublishedEntity & {
   entityId: number
   entityPath: string
   administration: Administration
+  breadcrumbs?: BreadcrumbItem[]
```

Comment:
the cms pr was merged, so it is providing breadcrumbs now on all envs, i think we should remove the `?`'s

https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/node/news_story